modules/web/client/src/main/scala/seqexec/web/client/components/tabs/TabsArea.scala

```diff
@@ -34,18 +31,16 @@ object TabsArea {
     .builder[Props]("TabsArea")
     .stateless
     .render_P(p =>
-      GridColumn(width = Sixteen, clazz = SeqexecStyles.sequencesArea)(
+      React.Fragment(
         SeqexecTabs(p.router),
         tabsConnect(x =>
           React
             .Fragment(
               x().toList.collect {
```

Comment:
I think we can `collect[VdomNode]` instead of the type ascriptions in each case.

Reply:
Cool, works nicely. I'll add it in the next PR